common/models/types/index.d.ts

```diff
@@ -238,6 +238,8 @@ declare interface Suggestion {
   tag?: SuggestionTag;
 }
 
+interface Reversion extends Suggestion { }
```

Comment:
Is this supposed to be the following?

```suggestion
interface Reversion extends Suggestion {
  tag: 'revert';
}
```

Reply:
I mean, I didn't realize it at the time... but yes, it _is_ supposed to be that.  Good idea.